src/cmd/create.go

```diff
@@ -418,8 +418,12 @@ func createContainer(container, image, release string, showCommandToEnter bool)
 
 	createArgs = append(createArgs, xdgRuntimeDirEnv...)
 
+	hostname, err := os.Hostname()
+	containerHostname := container + "." + hostname
```

Comment:
For the purpose of implementing https://github.com/containers/toolbox/issues/969 let's use `toolbox.<host's hostname>` as the container's hostname.  We might still have to revisit the `toolbox` prefix, but it will still be an improvement.

The [issue](https://github.com/containers/toolbox/issues/98) about making it easier to distinguish one Toolbx container from another is different from https://github.com/containers/toolbox/issues/969  It's already possible today to use `/run/.containerenv` and `/run/.toolboxenv` to come up with one's own shell prompt without making any changes to the Toolbx code.  Having said that, it will definitely be good to improve the default prompt that Toolbx provides, but let's not worry about that in this pull request.

Baby steps.  :)